typescript/sdk/src/deploy/core/HyperlaneCoreChecker.ts

```diff
@@ -68,6 +83,78 @@ export class HyperlaneCoreChecker<
     }
   }
 
+  async checkBytecodeHashes(chain: Chain): Promise<void> {
```

Comment:
Do you have a sense of how these violations would be remediated? In other words, is the goal here just to flag these violations? Or to fix them?

Reply:
I guess I'm just not totally clear what the point of this is

Reply:
Added a comment to that effect on the method